frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -43,5 +43,18 @@ export function useFilteredFacets(productList: ProductList) {
       return usefulFacets;
    }, [facets, infoNames, isItemTypeProductList]);
 
+   if (productList.type === ProductListType.AllTools) {
```

Comment:
I think this code may be right, but the values behind this may be incorrect.

See the strapi enum:
https://github.com/iFixit/react-commerce/blob/7c857922613b241a07ecc542dcb4e6c8296a81c1/frontend/lib/strapi-sdk/generated/sdk.ts#L216-L222

And our own enum:
https://github.com/iFixit/react-commerce/blob/7c857922613b241a07ecc542dcb4e6c8296a81c1/frontend/models/product-list/types.ts#L27-L34

It appears there's a mapping function too:
https://github.com/iFixit/react-commerce/blob/7c857922613b241a07ecc542dcb4e6c8296a81c1/frontend/models/product-list/index.ts#L151-L161

Reply:
Hey @danielbeardsley, `Enum_Productlist_Type` can be confusing indeed. Ideally I would have liked to define the enum in Strapi like this:

```ts
export enum Enum_Productlist_Type { 
  AllParts = 'all_parts', 
  AllTools = 'all_tools', 
  Marketing = 'marketing', 
  DeviceParts = 'device_parts', 
  ToolsCategory = 'tools_category', 
} 
```

Since most `device_parts` product list have the type already set to `parts` in production Strapi. I didn't want to introduce a breaking change. If we want to improve this, we need also to coordinate with the integration that imports product lists on Strapi.